modules/calc/calc.block.ini

```diff
@@ -50,7 +50,7 @@ description: Scale divisor after add
 [SHIFT]
```

Comment:
Don't forget to remove the `[FUNC]` block as well; @coretl tells me it's a relic of an earlier version.

Reply:
done